### PR TITLE
1 14 dev sync lrk

### PR DESCRIPTION
**Pull request by lrknox**

### CODE REVIEW

File: .github/workflows/aocc-auto.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
+name: hdf5 dev PAR autotools aocc ompi
```
Comment (byrnHDF) [inline .github/workflows/aocc-auto.yml]: dev to 1.14

File: .github/workflows/cmake-ctest.yml [hunk 2]
```diff
@@ -425,6 +464,15 @@ jobs:
           Copy-Item -Path ${{ runner.workspace }}/hdf5/build114/${{ inputs.preset_name }}-Intel/README.md -Destination ${{ runner.workspace }}/build114/hdf5/
           Copy-Item -Path ${{ runner.workspace }}/hdf5/build114/${{ inputs.preset_name }}-Intel/* -Destination ${{ runner.workspace }}/build114/hdf5/ -Include *.zip
           cd "${{ runner.workspace }}/build114"
+=======
```
Comment (byrnHDF) [inline .github/workflows/cmake-ctest.yml]: conflict comments

**Review by byrnHDF — APPROVED** [review]

File: .github/workflows/cmake-ctest.yml [hunk 3]
```diff
@@ -157,8 +176,8 @@ jobs:
         run: |
           mkdir "${{ runner.workspace }}/builddeb"
           mkdir "${{ runner.workspace }}/builddeb/hdf5"
-          cp ${{ runner.workspace }}/hdf5/hdfsrc/COPYING ${{ runner.workspace }}/builddeb/hdf5
-          cp ${{ runner.workspace }}/hdf5/hdfsrc/COPYING_LBNL_HDF5 ${{ runner.workspace }}/builddeb/hdf5
+          cp ${{ runner.workspace }}/hdf5/${{ steps.set-file-base.outputs.SOURCE_BASE }}/COPYING ${{ runner.workspace }}/builddeb/hdf5
+          cp ${{ runner.workspace }}/hdf5/${{ steps.set-file-base.outputs.SOURCE_BASE }}/COPYING_LBNL_HDF5 ${{ runner.workspace }}/builddeb/hdf5
           cp ${{ runner.workspace }}/hdf5/build/${{ inputs.preset_name }}-GNUC/README.md ${{ runner.workspace }}/builddeb/hdf5
```
Comment (byrnHDF) [inline .github/workflows/cmake-ctest.yml]: build to build114

File: .github/workflows/cmake-ctest.yml [hunk 4]
```diff
@@ -170,8 +189,8 @@ jobs:
         run: |
           mkdir "${{ runner.workspace }}/buildrpm"
           mkdir "${{ runner.workspace }}/buildrpm/hdf5"
-          cp ${{ runner.workspace }}/hdf5/hdfsrc/COPYING ${{ runner.workspace }}/buildrpm/hdf5
-          cp ${{ runner.workspace }}/hdf5/hdfsrc/COPYING_LBNL_HDF5 ${{ runner.workspace }}/buildrpm/hdf5
+          cp ${{ runner.workspace }}/hdf5/${{ steps.set-file-base.outputs.SOURCE_BASE }}/COPYING ${{ runner.workspace }}/buildrpm/hdf5
+          cp ${{ runner.workspace }}/hdf5/${{ steps.set-file-base.outputs.SOURCE_BASE }}/COPYING_LBNL_HDF5 ${{ runner.workspace }}/buildrpm/hdf5
           cp ${{ runner.workspace }}/hdf5/build/${{ inputs.preset_name }}-GNUC/README.md ${{ runner.workspace }}/buildrpm/hdf5
```
Comment (byrnHDF) [inline .github/workflows/cmake-ctest.yml]: build to build114

**Review by derobins — APPROVED** [review]

File: .github/workflows/cmake-ctest.yml (hunk 4 above)
Comment (lrknox) [inline .github/workflows/cmake-ctest.yml:194]:
```suggestion
          cp ${{ runner.workspace }}/hdf5/build/${{ inputs.preset_name }}-GNUC/README.md ${{ runner.workspace }}/buildrpm/hdf5
```
```suggestion
          cp ${{ runner.workspace }}/hdf5/build114/${{ inputs.preset_name }}-GNUC/README.md ${{ runner.workspace }}/buildrpm/hdf5
          cp ${{ runner.workspace }}/hdf5/build114/${{ inputs.preset_name }}-GNUC/*.rpm ${{ runner.workspace }}/buildrpm/hdf5
```

**Pull request merged**